src/OrchardCore/Orchard.ContentManagement/Orchard.ContentManagement.csproj

```diff
@@ -13,11 +13,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Html.Abstractions" Version="1.1.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.DataAnnotations" Version="1.1.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.ViewFeatures" Version="1.1.2" />
-    <PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="1.1.1" />
-    <PackageReference Include="YesSql.Abstractions" Version="2.0.0-beta-1026" />
+    <PackageReference Include="Microsoft.AspNetCore.Html.Abstractions" Version="1.1.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.DataAnnotations" Version="1.1.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.ViewFeatures" Version="1.1.3" />
+    <PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="1.1.2" />
+    <PackageReference Include="YesSql.Abstractions" Version="2.0.0-beta-1035" />
```

Comment:
This `Microsoft.Extensions.DependencyModel`: we used to reference an old version on purpose, like from a year ago as we were using something that was removed. However since we are not doing dynamic compilation anymore I thought we'd not need anything special. Maybe unrelated though.